crates/turbopack-ecmascript/src/analyzer/well_known.rs

```diff
@@ -337,6 +345,99 @@ pub fn require(args: Vec<JsValue>) -> JsValue {
     }
 }
 
+pub fn require_context_require(val: RequireContextValue, args: Vec<JsValue>) -> JsValue {
+    if args.len() < 1 {
+        return JsValue::Unknown(
+            Some(Arc::new(JsValue::call(
+                box JsValue::WellKnownFunction(WellKnownFunctionKind::RequireContextRequire(val)),
+                args,
+            ))),
+            "only a single argument is supported",
+        );
+    }
+
+    let Some(s) = args[0].as_str() else {
+        return JsValue::Unknown(
+            Some(Arc::new(JsValue::call(
+                box JsValue::WellKnownFunction(WellKnownFunctionKind::RequireContextRequire(val)),
+                args,
+            ))),
+            "only constant argument is supported",
+        );
+    };
+
+    let Some(m) = val.map.get(s) else {
+       return JsValue::Unknown(
+           Some(Arc::new(JsValue::call(
+               box JsValue::WellKnownFunction(WellKnownFunctionKind::RequireContextRequire(val)),
+               args,
+           ))),
+           "argument has to be in the context",
+       );
+    };
+
+    JsValue::Module(ModuleValue {
+        module: m.to_string().into(),
+        annotations: ImportAnnotations::default(),
+    })
+}
+
+pub fn require_context_require_keys(val: RequireContextValue, args: Vec<JsValue>) -> JsValue {
+    if args.len() == 0 {
```

Comment:
```suggestion
    if args.is_empty() {
```

I'm surprised Clippy did not complain?